sale_line_refund_to_invoice_qty/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Jordi Masvidal <jordi.masvidal@forgeflow.com>
```

Comment:
I think you should be added as a contributor.

Reply:
Added.